ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1249,6 +1249,24 @@ FCU_CHWZTC2X_HT2XZTC2X_DFVSC2X:
   - CHWZTC2X
   - HT2XZTC2X
   - DFVSC2X
+  
+FCU_DFVSMC_CHWBRC_STM_RTM_FDPM_ZTC_SRC:
+  description: "FCU with Status, zone control, schedule run"
+  is_canonical: false
```

Comment:
Why could this not be canonical?

Reply:
no reason sir, was just copied from the other before it hehe. I'll change it.